### ISSUE TITLE
'Stop downloading after Nth oldest account' needed

### ISSUE THREAD

**Issue by PascalSoftwares** [AssetKid/raider-release#92]
Regardless of interval timer(currently Mass Update Sleep = 6min), instagram blocks login account after downloading multiple accounts.

Currently avoiding account block by manually stopping 'update all' after a few tens of updates and waiting hours to update again,. But always taking critical risk of account block when forgot to stop at proper moment.

'Update all' is practically an account nullifier... Feature(variable) that stops updating after certain numbers is needed.

**Comment by bart4371** [comment]
I have found for myself that IG is ok with me updating once every ~15 hours with 90-100 sec sleep. If I run the mass update more often, I get blocked. There appear to be various lists of accounts that IG has different, tougher restrictions on, perhaps you are on one which I haven't encountered yet.

**Comment by PascalSoftwares** [comment]
Yes, IG is constantly testing their algorithm. in theory every Raider approach is potential target of block to them. Safety measures are never enough.

> I have found for myself that IG is ok with me updating once every ~15 hours with 90-100 sec sleep. If I run the mass update more often, I get blocked. There appear to be various lists of accounts that IG has different, tougher restrictions on, perhaps you are on one which I haven't encountered yet.
